packages/core/src/elements/Table/interfaces.ts

```diff
@@ -38,6 +38,7 @@ export interface TableProps<T extends object = {}> {
   tableHeader?: boolean;
 
   data: T[];
+  // totalData: number;
```

Comment:
what about totalData?  for auto pagination

Reply:
Do you mean the infinity scrolling?

Reply:
No. Never mind. I need to fix something in FeTable before